ssl/statem/extensions_srvr.c

```diff
@@ -692,7 +692,8 @@ int tls_parse_ctos_cookie(SSL *s, PACKET *pkt, unsigned int context, X509 *x,
     unsigned char hmac[SHA256_DIGEST_LENGTH];
     unsigned char hrr[MAX_HRR_SIZE];
     size_t rawlen, hmaclen, hrrlen, ciphlen;
-    unsigned long tm, now;
+    time_t now;
+    unsigned long tm;
```

Comment:
`PACKET_get_net_4()` has an `unsigned long` argument, which can be either 32-bit or 64-bit depending on the system; so a temporary variable is needed. That function (and related functions) should probably be changed to always use a sized-specified (e.g. 32-bit) type. A temporary is needed regardless as `time_t` could be 64-bit or 32-bit.